sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -117,11 +117,20 @@ static uint8_t all_certs[8192];
 static size_t cdi_0_offset;
 static size_t cdi_1_offset;
 static cert_flash_info_layout_t cert_flash_layout[] = {
+    {
+        // The DICE UDS cert is placed on this page since it must remain stable
+        // post manufacturing. This page should never be erased by ROM_EXT, nor
+        // owner firmware.
+        .used = true,
+        .group_name = "FACTORY",
```

Comment:
could also be called DICE_UDS or some such

Reply:
I thought factory would be more encompasing in case we add other factory endorsed certs to the base perso firmware